mono/mini/interp/interp.c

```diff
@@ -461,9 +464,13 @@ get_context (void)
 		/*
 		 * Use two stacks, one for InterpFrame structures, one for data.
 		 * This is useful because InterpFrame structures don't need to be GC tracked.
+		 * It also useful to fit allocations into iframe_stack perfectly.
+		 * FIXME: The behavior when allocations do not fit in current data_stack
+		 * chunk is very poor. Attempt a scheme with one growable chunk instead
+		 * using multiple adjacent alloca.
 		 */
-		frame_stack_init (&context->iframe_stack, 8192);
-		frame_stack_init (&context->data_stack, 8192);
+		frame_stack_init (&context->iframe_stack, 1024 * sizeof (InterpFrame));
+		frame_stack_init (&context->data_stack, 65536);
```

Comment:
Is this always supposed to match `frag_size`? If so, that should be noted (or the value just pushed off into a constant somewhere).

Reply:
It doesn't have to, and does not actually (for iframe_stack).
It is initial allocation and later allocation.
I am a little bothered by iframe_stack later allocations maybe not being multiple of InterpFrame size, but 1024 frames is a lot.

Reply:
The initial allocations could remain small. The incremental ones are the bigger problem. Or just one large allocation. Could also try smaller upon failure.
But constant 4K increment is terrible.

Reply:
I couldn’t decide what to name the constant but yeah. Static const int, no need for define.